### DISCUSSION TITLE
Supply pod rarity

### DISCUSSION THREAD

**Discussion by S-W-N** [induktio/thinker#44] [General]
I am getting rather erratic results with no unity scattering + rare supply pods.

I got 11 supply pods near my start, which means I can't rely on rare being rare here, which made me test the map generation a bit.

A few tiny maps: Starts tend to have 0-3 supply pods or such.
Large maps: Often 0-3 or a bit more, but in some cases around 11-13 max. Maps that have a lot for one player tend to have a lot for a bunch of players, like 13 for one and 7-10 for half of the others, and then some have 0.
Very huge maps: Somewhat similar results as with large maps.

To some extent it seems like dry starts generate more pods as a compensation.

If it weren't for the compensation aspect, just "no supply pods" would be a nice setting.

**Comment by induktio** [comment]
Now that I think of it, no unity scattering + rare supply pods seems to alter the frequency near the starting site from normal amounts, so it can have lots of pods, but 11-13 sounds quite a lot. The distribution still uses the vanilla placement algorithm, except the frequency is lowered to use the sea pods density on land also.

Thinker uses a custom faction spawn algorithm and it does take the dryness into account when choosing the location, so that observation is certainly true. Tuning the weights for different factors is somewhat tricky though. Maybe it would make sense to add an extra option for rare_supply_pods so it would disable them altogether from the map. The game options seem to assume there's always pods at least near the landing site.
